docs/TELEMETRY.md

```diff
@@ -0,0 +1,131 @@
+# Telemetry system
+
+In Web Chat 4.8, telemetry system is introduced. Developers can opt into this feature by implementing their own telemetry collection adapter and start pumping measurements into telemetry services of their choice.
+
+Microsoft do not collect or receive any telemetry measurements for Web Chat, on both CDN and NPM releases. Developers must provide their own telemetry collection adapter to start collecting data. Under local regulations, they may be required to provide privacy policy to the end-user to explain their data collection policy.
+
+## Measurements
+
+We simplified our measurements to make it suitable for popular telemetry services:
+
+-  Dimensions are environment information, including browser capabilities and options passed to Web Chat
+-  Events will include a name, optional string data, and optional non-negative finite integer value
+-  Exceptions will include the JavaScript error object
+-  Timings will fire two events, `timingstart` and `timingend`
+   -  `timingstart` will include a name
+   -  `timingend` will include a name, and duration measured in milliseconds
```

Comment:
Can we change the format here to be:
`- Dimensions: dimensions are environment information ...`

Makes it a little easier to understand that we are providing definitions.